templates/hotgraphicPopup.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { useEffect } from 'react';
+import a11y from 'core/js/a11y';
+import { classes, templates } from 'core/js/reactHelpers';
+
+export default function HotgraphicPopup(props) {
+
+  const {
+    _items,
+    _isRound
+  } = props;
+
+  useEffect(() => {
+    const activeItem = _items.filter(item => item._isActive);
+
+    if (!activeItem.length) return;
+
+    const activeItemIndex = activeItem[0]._index;
+    const focusElement = $(`.hotgraphic-popup__item[data-index=${activeItemIndex}]`);
+
+    a11y.focusFirst(focusElement);
+  });
+
+  return (
+    <div className='hotgraphic-popup__inner'>
+
+      <templates.hotgraphicPopupToolbar {...props} />
```

Comment:
I think this should be moved to below the content to allow screen readers to jump directly to content rather than the toolbar.

Reply:
Agreed. Done.